Makefile

```diff
@@ -51,18 +55,43 @@ test-container-image: build ## Target to test the final image
 
 test: test-app test-container-image
 
-dev-reconcile-loop: build ## Trigger the reconcile loop inside a container for an integration
-	@$(CONTAINER_ENGINE) run --rm \
-		--add-host=host.docker.internal:host-gateway \
-		-v $(CURDIR):/work \
-		-e INTEGRATION_NAME=$(INTEGRATION_NAME) \
-		-e INTEGRATION_EXTRA_ARGS=$(INTEGRATION_EXTRA_ARGS) \
-		-e SLEEP_DURATION_SECS=$(SLEEP_DURATION_SECS) \
-		-e DRY_RUN=$(DRY_RUN) \
-		-e CONFIG=/work/config.dev.toml \
-		$(IMAGE_NAME):$(IMAGE_TAG)
-
 clean:
 	@rm -rf .tox .eggs reconcile.egg-info build .pytest_cache
 	@find . -name "__pycache__" -type d -print0 | xargs -0 rm -rf
 	@find . -name "*.pyc" -delete
+
+dev-clean:
+	@$(DEV_CONF_EXISTS)
+	$(DEV_CONF) docker-compose down
+
+dev-venv: ## Create a local venv for your IDE and remote debugging
+	rm -rf venv
```

Comment:
Shouldn't this be part of either `clean` or `dev-clean`?

Reply:
Moved it to `clean` now